xla/service/gpu/tests/reduce_column_layout_change.hlo

```diff
@@ -160,11 +173,11 @@ ENTRY kernel_entry {
 // CHECK:         %[[VAL_94:.*]] = mul nuw nsw i32 %[[VAL_73]], 1
 // CHECK:         %[[VAL_95:.*]] = add nuw nsw i32 0, %[[VAL_94]]
 // CHECK:         %[[VAL_96:.*]] = getelementptr inbounds [12 x [3 x [32 x [16 x [32 x [4 x [3 x [12 x float]]]]]]]], ptr %[[VAL_97:.*]], i32 0, i32 %[[VAL_92]], i32 %[[VAL_91]], i32 %[[VAL_89]], i32 %[[VAL_85]], i32 %[[VAL_84]], i32 %[[VAL_82]], i32 %[[VAL_80]], i32 %[[VAL_78]]
-// CHECK:         %[[VAL_98:.*]] = load float, ptr %[[VAL_96]], align 4, !invariant.load !9
-// CHECK:         store float %[[VAL_98]], ptr %[[VAL_14]], align 4
-// CHECK:         call void @[[REDUCTION0]](ptr %[[VAL_13]], ptr %[[VAL_14]], ptr %[[VAL_10]])
-// CHECK:         %[[VAL_99:.*]] = load float, ptr %[[VAL_10]], align 4
-// CHECK:         store float %[[VAL_99]], ptr %[[VAL_13]], align 4
+// CHECK:         %[[VAL_98:.*]] = load float, ptr{{( addrspace\(5\))?}} %[[VAL_96]], align 4, !invariant.load !{{[0-9]}}
```

Comment:
maybe separate lines CHECK-GCN and CHECK-PTX?

Reply:
if I do separate lines CHECK-GCN and CHECK-PTX for every load and store, it will mean a lot of code duplication